File: submissions/sm_102_amit/week_03/day_3/session_2/split_string.js
// split string
function splitString(input, point) {
  input = input.toLowerCase();
  point = point.toLowerCase();
  var array = [];
  for (var i = 0; i < input.length; i++) {
    if (input[i] == point) {
      array.push(" ");
    } else {
      array.push(input[i]);
    }
  }

  console.log(array.join(""));
}

splitString("fdfdamit Kumar jkdshfa asd", "a");

File: submissions/sm_026_rohit-kumar/week_13/day_4/session_1/split_string.py
string = input('Enter string : ')
char = input('Enter split char : ').split()[0]


temp_str = ''
lst = []
for c in string:
    if c != char:
        temp_str = temp_str + c
    else:
        lst.append(temp_str)
        temp_str = ''

lst.append(temp_str)
print(lst)
